Dockerfile

```diff
@@ -14,21 +14,26 @@ EXPOSE 8000
 
 # Install operating system dependencies.
 RUN apt-get update -y && \
-    apt-get install -y apt-transport-https rsync libmagickwand-dev && \
+    apt-get install -y apt-transport-https rsync libmagickwand-dev unzip postgresql-client-13 && \
```

Comment:
`postgresql-client-13` doesn't seem to be a valid package name here, I get this error:

```
Step 5/25 : RUN apt-get update -y &&     apt-get install -y apt-transport-https rsync libmagickwand-dev unzip postgresql-client-13 &&     rm -rf /var/lib/apt/lists/*
 ---> Running in e83a2c1b5f63
Get:1 http://deb.debian.org/debian buster InRelease [122 kB]
Get:2 http://security.debian.org/debian-security buster/updates InRelease [65.4 kB]
Get:3 http://deb.debian.org/debian buster-updates InRelease [51.9 kB]
Get:4 http://security.debian.org/debian-security buster/updates/main amd64 Packages [311 kB]
Get:5 http://deb.debian.org/debian buster/main amd64 Packages [7906 kB]
Get:6 http://deb.debian.org/debian buster-updates/main amd64 Packages [15.5 kB]
Fetched 8471 kB in 3s (2833 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package postgresql-client-13
The command '/bin/sh -c apt-get update -y &&     apt-get install -y apt-transport-https rsync libmagickwand-dev unzip postgresql-client-13 &&     rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
ERROR: Service 'web' failed to build
make[1]: *** [Makefile:9: rebuild] Error 1
make[1]: Leaving directory '/home/bdickinson/Git/wagtail.io'
make: *** [Makefile:4: setup] Error 2
```

Reply:
oh, interesting, I wonder if you are getting a different image for ``python:3.8`` to me?
```
python                                                      3.8                 ce9555db0df5   4 weeks ago         909MB
```

Reply:
I think my local `3.8` tag must have been something `buster`-ish. I pulled a new image and it started working again. I'd suggest changing the base image to `python:3.8-bullseye` to fix this!

Reply:
Done, thanks!